python/pyarrow/pandas_compat.py

```diff
@@ -621,13 +620,12 @@ def convert_column(col, field):
         except (pa.ArrowInvalid,
                 pa.ArrowNotImplementedError,
                 pa.ArrowTypeError) as e:
-            e.args += ("Conversion failed for column {!s} with type {!s}"
-                       .format(col.name, col.dtype),)
+            e.args += (
+                f"Conversion failed for column {col.name!s} with type {col.dtype!s}",)
```

Comment:
Same here: `!s` is probably unnecessary.

Reply:
Resolved in [c926dde](https://github.com/apache/arrow/pull/45629/commits/c926ddec73bfdeb7d6c3b3768ade16f8146eadd4)